tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -648,6 +648,18 @@ def testEllipsis(self):
       with self.assertRaisesRegexp(ValueError, "Multiple ellipses"):
         _ = checker[..., :, ...].eval()
 
+  @test_util.run_deprecated_v1
```

Comment:
Can this run_in_graph_and_eager_modes for more coverage instead of deprecated_v1?

Reply:
I could add more test here, could you please explain the difference between these two?

Reply:
This is currently only running the test in graph mode; eager mode slicing is equally important.

See other tests with run_in_graph_and_eager_modes and read the documentation of that and this decorator to understand the difference

Reply:
@alextp, Thanks for your explanation, From my example below, I think it could run_in_graph_and_eager_modes, should I add a test here?

Reply:
Yes